CHANGELOG.md

```diff
@@ -240,7 +242,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - The first tagged version.
 
-[Unreleased]: https://github.com/cyberark/conjur/compare/v1.4.3...HEAD
+[Unreleased]: https://github.com/cyberark/conjur/compare/v1.4.4...HEAD
```

Comment:
shouldn't the head be `1.4.5...HEAD`? 
The 1.4.4 one was 
```
[Unreleased]: https://github.com/cyberark/conjur/compare/v1.4.4...HEAD
[1.4.4]: https://github.com/cyberark/conjur/compare/v1.4.3...v1.4.4
```
https://github.com/cyberark/conjur/pull/1251/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL219

Reply:
correct. fixed it.